kong/clustering/control_plane.lua

```diff
@@ -173,8 +173,7 @@ function _M:push_config()
     n = n + 1
   end
 
-  ngx_update_time()
-  local duration = ngx_now() - start
+  local duration = now_updated() - start
```

Comment:
I mean this.

Reply:
I searched the repo by `git grep 'now_updated() -'`, only found two places, perhaps it is not the time to add this function.

Reply:
`- start` reveals a lot more. And there may be many more too.

Reply:
but we can do it in other pr for sure, if we decide to do it.

Reply:
Yes, Let's leave it to another PR, keep this PR focus on one topic.